recipes/encyclopedia/encyclopedia.py

```diff
@@ -14,8 +14,8 @@
 from os import getenv
 from os import X_OK
 
-# https://bitbucket.org/searleb/encyclopedia/downloads/encyclopedia-0.9.5-executable.jar
-jar_file = 'encyclopedia-0.9.5-executable.jar'
+# https://bitbucket.org/searleb/encyclopedia/downloads/encyclopedia-1.2.2-executable.jar
+jar_file = 'encyclopedia-1.2.2-executable.jar'
```

Comment:
I wonder if this would eliminate the need to manually update this file with every new version...
```suggestion
jar_file = f'encyclopedia-{ getenv("PKG_VERSION") }-executable.jar'
```

Reply:
@fanninpm   I'll test that.  Thanks.

Reply:
If this works, then BiocondaBot will be able to update this recipe, hopefully without any manual intervention.